### PR TITLE
add 'slow' jenkins e2e back into suite (e2e-aws-jenkins only, not e2e…

### PR DESCRIPTION
**Pull request by gabemontero**
…-aws-builds)

### CODE REVIEW

**Comment by gabemontero** [comment]
https://github.com/openshift/release/pull/4085 is adding e2e-aws-jenkins to openshift/origin jobs

File: test/extended/builds/pipeline_jenkins_e2e.go [hunk 1]
```diff
@@ -454,7 +452,7 @@ var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow samples pipelin
 				o.Expect(envs[0].Value).To(o.Equal("BAR1"))
 			})
 
-			g.By("Blue-green pipeline")
+			/*g.By("Blue-green pipeline")
```
Comment (adambkaplan) [inline test/extended/builds/pipeline_jenkins_e2e.go]: Are we going to revert this?

File: test/extended/builds/pipeline_jenkins_e2e.go [hunk 2]
```diff
@@ -24,7 +22,7 @@ import (
 	"github.com/openshift/origin/test/extended/util/jenkins"
 )
 
-var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow samples pipeline build", func() {
+var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow openshift pipeline build", func() {
```
Comment (adambkaplan) [inline test/extended/builds/pipeline_jenkins_e2e.go]: Don't we have a `[Feature:Jenkins]` tag?

File: test/extended/builds/pipeline_jenkins_e2e.go (hunk 1 above)
Comment (gabemontero) [reply test/extended/builds/pipeline_jenkins_e2e.go]: either revert or delete entirely

I'm going to try some runs with both and see how flake this test still is.  Last time I tried it was flaky but that was a couple of months ago.

File: test/extended/builds/pipeline_jenkins_e2e.go (hunk 2 above)
Comment (gabemontero) [reply test/extended/builds/pipeline_jenkins_e2e.go]: No we just use "openshift pipeline" as the ginkgo focus in `openshift-tests` .... I vaguely remember way back when there was an issue with multiple `Feature` directives (and the pipeline_origin.go has `[Feature:Builds]` ) but I can experiment with such a switch as well once I narrow down which tests are reliable.

**Comment by gabemontero** [comment]
ok the e2e-aws-jenkins run as has started the tests and they have picked up the additional tests:

```
gmontero ~ $ oc logs -f e2e-aws-jenkins -c test
secret/support created
started: (0/1/4) "[Slow]jenkins repos e2e openshift using slow openshift pipeline build Sync plugin tests using the persistent template [Suite:openshift]"

started: (0/2/4) "[Feature:Builds][Slow] openshift pipeline build Sync plugin tests using the ephemeral template [Suite:openshift]"

started: (0/3/4) "[Feature:Builds][Slow] openshift pipeline build jenkins-client-plugin tests using the ephemeral template [Suite:openshift]"

```

note we only run 3 at a time given Jenkins is a memory hog

**Comment by gabemontero** [comment]
one passing test ... let's try another

/test e2e-aws-jenkins

**Comment by gabemontero** [comment]
another pass ... one more time

/test e2e-aws-jenkins

**Comment by gabemontero** [comment]
ok added the bluegreen pipeline back in .... let's get a few runs in, see what is up

**Comment by gabemontero** [comment]
note the e2e-aws-jenkins job was NOT started by default when I updated the commits ... GOOD

**Comment by gabemontero** [comment]
initial run with bluegreen looks good ... trying another 

/test e2e-aws-jenkins

**Comment by gabemontero** [comment]
Also, switched the jenkins e2e ginkgo focus to `[Feature:Jenkins]` and things like good in my local runs for both e2e-aws-builds and e2e-aws-jenkins wrt which tests get run

I'll push that changes after the next test run completes @adambkaplan and remove the hold so you can add additional comments / approve as you deem appropriate

**Comment by gabemontero** [comment]
artifacts are still gathering but ` 2019/06/21 18:11:59 Container test in pod e2e-aws-jenkins completed successfully `

**Comment by gabemontero** [comment]
/hold cancel

/assign @adambkaplan 

ready for "official" review

**Comment by gabemontero** [comment]
/assign @pweil- 

you are in the cmd/OWNERS approver list :-) .... per suggestion from @adambkaplan I've updated the ginkgo focus for the jenkins e2e suite to `[Feature:Jenkins]`

**Comment by gabemontero** [comment]
thanks @bparees

**Pull request merged**